website/docs/configuration.md

```diff
@@ -526,6 +528,50 @@ Group non-emtpy consecutive layers (zindex) together, so that the shadows and bl
 the whole group instead of each individual layer. This can get rid of some shadowing and blending
 artifacts, but cause worse problems like [#2574](https://github.com/neovide/neovide/issues/2574).
 
+#### Box Drawing
+
+The unicode standard defines several code points that are useful to draw [boxes, diagrams or are
+otherwise decorations][https://en.wikipedia.org/wiki/Box_Drawing]. A font file can include graphical
+representation for several of these code points (glyphs). For example, [Nerd
+Fonts][https://www.nerdfonts.com/] is a collection of font faces that have been patched to include
+glyphs for several box drawing code points (and many other use-cases).
+
+When Neovide renders these glyphs, some glyphs might not line up correctly or might have gaps
+between adjacent cells, breaking visual continuity. This is especially pronounced when using the
+[linespace][#line-spacing] configuration option to add spacing between lines.
+
+Neovide has support for native rendering (i.e ignore the glyph data in the font) for a subset of
+these glyphs to avoid this problem. You can configure this via:
+
+```config.toml
+[box-drawing]
+# "font-glyph", "native" or "selected-native"
+mode = "native"
+# selected = "🮐🮑🮒"
+```
+
+- `font-glyph` uses the glyph data in the font file.
+- `native` (default) turns on native rendering for all supported box drawing glyphs.
+- `selected-native` turns on native rendering for only code points specified in the `selected`
+  setting.
+
+The width of the lines drawn can be further controlled using the following settings (defaults shown):
+
+```toml
+[box-drawing]
+stroke-width-ratio = 0.15
+thickness-multipliers = [1.0, 1.5, 2.0]
```

Comment:
I think it's probably best to specify this in pixels. I'm not sure that I would like it to change with the font size, at least not linearly.

Reply:
I went back and forth on this. When specified in absolute pixel size, when zooming in using the scale factor, the lines being super thin compared to the text looked weird (to my eyes). I generally like it when applications scale the entire interface instead of just the text size.

I don't have a strong opinion on this though.